lacci/test/test_oval.rb

```diff
@@ -26,6 +26,7 @@ def test_oval_with_height
       end
     SHOES_APP
       ov = oval()
+      binding.irb
```

Comment:
Was this on purpose?

Reply:
No. No it is not. Thanks noah, this was a bigger pr

Reply:
and I may have pushed more quickly as I'm cooking